go.mod

```diff
@@ -29,11 +29,11 @@ require (
 	k8s.io/client-go v0.31.0
 	knative.dev/eventing v0.43.3
 	knative.dev/eventing-kafka-broker v0.37.0
-	knative.dev/hack v0.0.0-20240814130635-06f7aff93954
+	knative.dev/hack v0.0.0-20250117112405-6cb0feb3ac46
 	knative.dev/networking v0.0.0-20240716111826-bab7f2a3e556
-	knative.dev/operator v0.42.5
+	knative.dev/operator v0.42.11-0.20250319103414-9c594ced1a3a
```

Comment:
You got this via:

```
go get -u knative.dev/operator@release-1.15
make generated-files
```

? 

Or different? (just wondering also b/c of the other bumps (hack/serving)

Reply:
Yep, exactly. The `knative/operator` has that https://github.com/knative/operator/blob/release-1.15/go.mod#L27. But we have a replace directive later anyway to use our midstreams. It's a cosmetic version change.